sdk/monitor/opentelemetry-exporter/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@azure/monitor-opentelemetry-exporter",
+  "author": "appinsightssdk@microsoft.com",
+  "version": "1.0.0-preview.4",
+  "description": "Application Insights exporter for the OpenTelemetry JavaScript (Node.js) SDK",
```

Comment:
does the exporter work in the browser too?

Reply:
It is not planned to work in the browser for now